index.js

```diff
@@ -34,26 +37,59 @@ app.get('/login', (req, res) => {
 });
 
 //post route for login form
+//change it with passport.js
 app.post('/home', (req, res) => {
```

Comment:
Does this need to have a path of '/login' instead of home?

Reply:
You are most likely right